web/src/components/dashboard/ProjectRequests.tsx

```diff
@@ -74,20 +74,22 @@ const ProjectRequests: React.FC<any> = (props) => {
       {
         Header: 'Response',
         accessor: 'profileId',
-        Cell: ({ row: { values } }: any) => (
-          <Box>
-            <button
-              type="button"
-              value={values.profileId}
-              onClick={() => {
-                setProfileId(values.profileId);
-                toggle();
-              }}
-            >
-              Review
-            </button>
-          </Box>
-        ),
+        Cell: ({ row: { values } }: any) =>
+          isAdminUser && (
+            <Box>
+              <button
+                type="button"
+                value={values.profileId}
+                onClick={() => {
+                  console.log('Click');
```

Comment:
please remove console.log( I suggest we can add no console rule later in our eslint rule)

Reply:
Yes thank you for catching that!